config/environments/production.rb

```diff
@@ -56,7 +56,7 @@
 
   # Enable locale fallbacks for I18n (makes lookups for any locale fall back to
   # the I18n.default_locale when a translation can not be found)
-  config.i18n.fallbacks = [:en]
+  config.i18n.fallbacks = true
```

Comment:
Won't this break the desired behaviour of falling back to `:en` when `default_locale` is missing a translation? In other words, shouldn't we keep this as `[:en]`? Or am I misunderstanding something?

If we can keep it as `[:en]`, can we then move it to `application.rb` and remove it from all of the individual environment config files?

Reply:
Bump @saimonmoore

Reply:
shit...that change shouldn't have made it into the PR. Good catch @oeoeaio. Not sure about KISSing it though. I think it's better to keep it in the inidividual files as I understand ofn may start to support multiple locales.

Reply:
👍  sounds good.